cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/profile/OAuthUserProfile.java

```diff
@@ -0,0 +1,14 @@
+package org.jasig.cas.support.oauth.profile;
+
+import org.pac4j.http.profile.HttpProfile;
+
+/**
+ * Specific profile for OAuth user authentication.
+ *
+ * @author Jerome Leleu
+ * @since 4.3.0
+ */
+public class OAuthUserProfile extends HttpProfile {
```

Comment:
Ok. In that case, why are we creating a new instance if it's exactly the same as HttpProfile? Seems like a waste, no? What does this profile give us that HttpProfile does not?